.devcontainer/adx/create-adx.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Display Help
+Help()
+{
+   echo "Create data explorer cluster for development purpose."
+   echo
+   echo "Syntax: create-adx.sh [-u|h]"
+   echo "options:"
+   echo "u     Unique name used to create resources."
+   echo "h     Print this Help."
+   echo
+}
+
+while getopts u:h flag;
+do
+    case "${flag}" in
+        h) Help
+           exit;;
+        u) unique=${OPTARG};;
+    esac
+done
+
+if [ ! "$unique" ]; then
+    echo "Please add unique name argument."
+    exit;
+fi
+
+# Resource Names
+RESOURCE_GROUP_LOCATION="westeurope"
```

Comment:
'location' might be a good option to be able to pass as arg

Reply:
Good point. I added the location argument.